python/src/nanoarrow/__init__.py

```diff
@@ -26,4 +26,40 @@
     allocate_c_array,
     allocate_c_array_stream,
 )
+from nanoarrow.schema import (  # noqa: F401
+    Schema,
+    Type,
+    TimeUnit,
+    schema,
+    null,
+    bool,
+    int8,
+    uint8,
+    int16,
+    uint16,
+    int32,
+    uint32,
+    int64,
+    uint64,
+    float16,
+    float32,
+    float64,
+    string,
+    large_string,
+    binary,
+    large_binary,
+    fixed_size_binary,
+    date32,
+    date64,
+    time32,
+    time64,
+    timestamp,
+    duration,
+    interval_months,
```

Comment:
Probably more of a pyarrow issue than something here but I don't think there is a pyarrow exposure for any interval aside from month/day/nano